src/app/commons/IndexBanner/InPageNavigationBar.js

```diff
@@ -20,15 +19,15 @@ function link(currentPathName, to, name, Icon) {
  *              (3) Icon, the Icon on the tab
  */
 
-const InPageNavigationBar = function ({currentPathName, path, tabContents}) {
+const InPageNavigationBar = function ({currentURL, path, tabContents}) {
     return (
         <div className="in-page-nav-bar">
             <p className="path">
-                <FakeLink content={path.head}/><span> / </span><FakeLink content={path.tail}/>
+                <Link to="/exams">{path.head}</Link><span> / </span>{path.tail}
             </p>
```

Comment:
1. You shouldn't link to `/exams` directly, this is a common component.
2. change it to `<Link to="/exams">{path.head}</Link><span> / </span><Link to="/exams">{path.tail}</Link>`. The `{path.tail}` is also a link. 
     In the exam scenario, when you click `{path.tail}` it will go to the exam problem page.

Reply:
@Chaoyuuu: If 1. is true then how can I satisfy 2.?

Reply:
I've updated for 1. and found this issue, since other usage might not have problem page?